compute/instances/troubleshooting/bootscript-eol.mdx

```diff
@@ -0,0 +1,78 @@
+---
+meta:
+  title: Dealing with the end of life of the bootscript feature
+  description: This page helps you troubleshoot issues related to the EOL of the bootscript feature
+content:
+  h1: Dealing with the end of life of the bootscript feature
+  paragraph: This page helps you troubleshoot issues related to the EOL of the bootscript feature
+tags: bootscript boot-script instance boot script
+dates:
+  validation: 2023-02-20
+  posted: 2023-02-20
+categories:
+  - compute
+---
+
+Certain older legacy Instance types (e.g. **VC1-x**, **X64-x**, **Start1-x**, and **Stardust**) proposed the option to use a bootscript (a preconfigured boot configuration) to start your Instances. This option has become deprecated and will no longer be supported.
+If you are still using one of these Instance types, you have to change your boot configuration in order to maintain your Instance operational.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created your SSH key and added it to your account](/console/my-project/how-to/create-ssh-key/)
+  - You have [an Instance](/compute/instances/how-to/create-an-instance) using a bootscript
+</Message>
+
+## Migration options for Instances using bootscripts
+
+### Instance using bootscript method and an old OS image without UEFI
+
+The following options apply to most of the existing **VC1-x**, **X64-x**, **Start1-x**, and **Stardust** Instances.
+
+<Message type="important">
+  Local volume(s) not supporting UEFI cannot be used to boot any other type of Instance. You have to retrieve your data and manually migrate it to a Instance supporting UEFI boot. There is no way on Scaleway side to identify which customer is using OS image with / without UEFI partition. Only the user can find out looking into his volume.
```

Comment:
```suggestion
  Local volume(s) not supporting UEFI cannot be used to boot any other type of Instance. You have to retrieve your data and manually migrate it to a Instance supporting UEFI boot. Unfortunately we cannot identify, on the Scaleway side, which customers are using OS images with / without UEFI partition. This can only be identified by the user themselves, looking into their volumes.
```